src/driver-holtek.h

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+/*
+ * Copyright © 2023 Michał Lubas
+ */
+
+#pragma once
+
+#include "libratbag-util.h"
+
+#define HOLTEK8_FW_VERSION_LEN 4
+
+enum holtek8_sensor {
+	HOTLEK8_SENSOR_UNKNOWN = 0,
+	HOTLEK8_SENSOR_PAW3333,
+	HOTLEK8_SENSOR_PMW3320,
```

Comment:
"HOTLEK" vs "HOLTEK", typo?

Reply:
I somehow missed it 44 times... Fixed :D